### DISCUSSION TITLE
How to disable specific rule for cpplint?

### DISCUSSION THREAD

**Discussion by simonppg** [super-linter/super-linter#2025] [Q&A] [answered]
I'm looking for a way to remove some cpplint checks.

I have a .editorconfig file with the following configuration:
```
root = true

[*]
indent_style = space
indent_size = 2
continuation_indent_size = 2
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
```

I added super linter to my project with the default .yml file provided on [the guide](https://github.com/marketplace/actions/super-linter). I worked awesome, but soon I found that one of my rules in `.editorconfig` file conflicts with `private: should be indented +1 space inside class` cpplint rule.

I want to keep my Editorconfig rule of 2 spaces, so I need to disable a cpplinter rule but I can't find a way to do it. could you please give me some clue? or point me to the right direction to do this?

**Comment by simonppg** [comment, marked as answer]
The solution was so easy, I just need to look carefully on the error message, it shows the name at the end of the message between "[]"

```shell
LinuxFilesManager.hpp:7:  private: should be indented +1 space inside class LinuxFilesManager  [whitespace/indent] [3]
```
The solution was creating the CPPLINT.cfg file like this:

```shell
set noparent

filter=-whitespace/indent
```

**Comment by Franciscoreis32** [comment]
Olá

**Comment by Franciscoreis32** [comment]
Boa tarde
